source/macos/AdaptiveCards/AdaptiveCards/Renderers/BaseCardElementRenderer.swift

```diff
@@ -20,11 +20,14 @@ class BaseCardElementRenderer {
             }
         }
         
-        if let collectionElement = element as? ACSCollectionTypeElement, let columnView = view as? ACRColumnView {
-            if let backgroundImage = collectionElement.getBackgroundImage(), let url = backgroundImage.getUrl() {
+        if let collectionElement = element as? ACSCollectionTypeElement {
+            if let columnView = view as? ACRColumnView, let backgroundImage = collectionElement.getBackgroundImage(), let url = backgroundImage.getUrl() {
                 columnView.setupBackgroundImageProperties(backgroundImage)
                 rootView.registerImageHandlingView(columnView.backgroundImageView, for: url)
             }
+            if let minHeight = collectionElement.getMinHeight(), let heightPt = CGFloat(exactly: minHeight), heightPt > 0 {
+                view.heightAnchor.constraint(greaterThanOrEqualToConstant: heightPt).isActive = true
+            }
```

Comment:
```suggestion
        if let collectionElement = element as? ACSCollectionTypeElement, let columnView = view as? ACRColumnView {
            if let backgroundImage = collectionElement.getBackgroundImage(), let url = backgroundImage.getUrl() {
                columnView.setupBackgroundImageProperties(backgroundImage)
                rootView.registerImageHandlingView(columnView.backgroundImageView, for: url)
            }
            columnView.setMinHeight(collectionElement.getMinHeight())
```
All the validations are handled inside `ACRColumnView` [here](https://github.com/rohshar6/AdaptiveCards/blob/d7136b43152d7c3b6c6c7c6efba3c6892f96a325/source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift#L122-L125)